README.md

```diff
@@ -1,2 +1,69 @@
 # doppler-docker-akamai-publish
 Docker image to help us to publish releases to Akamai CDN
+
+# How to use cdn-uploader.js library:
+
+node cdn-uploader.js arg1 arg2
+
+* arg1: [REQUIRED] Is the base path to load different projects to the cdn, must be unique for each project.
+
+* arg2: [REQUIRED] Is the build path where you found your resources.
+
+Note: The shell context init in the folder than cdn-uploader.js is running.
+```bash
+.
+..
+cdn-uploader.js
+example/
+projects/other/stuff
+
+node cdn-uploader.js feature1 example   <-- This will upload the data inside of "example" folder recursively, note that the "example" folder will not be uploaded.
+
+node cdn-uploader.js feature2 projects/other <-- This will upload the data inside of "other" folder recursively, note that the "other" folder will not be uploaded but "stuff" yes.
+```
+
+CDN Url result:
+http://cdn.example.akamai.com/{arg1}/{arg2}/scripts/example.js
+
+http://cdn.example.akamai.com/{arg1}/{arg2}/images/example.gif
+
+Real examples:
+http://cdn.example.akamai.com/translator/buildV1/scripts/example.js
+
+http://cdn.example.akamai.com/dashboard/buildV2/images/example.gif
+
+
+We use docker-compose configuration .yml to define environment variables requested in:
+
+```js
+const config = {
+	host: process.env.AKAMAI_CDN_HOSTNAME,
+	user: process.env.AKAMAI_CDN_USERNAME,
+	password: process.env.AKAMAI_CDN_PASSWORD,
+	cpCode: process.env.AKAMAI_CDN_CPCODE,
+    port: 21,
+```
+}
```

Comment:
Maybe it is better to have an example of _.yml_ and not about where is being used.